src/components/date/__internal__/date-picker/date-picker.component.tsx

```diff
@@ -245,9 +238,6 @@ export const DatePicker = ({
             locale={{
               localize: {
                 ...defaultLocale.localize,
-                months: monthsLong,
```

Comment:
I really am not sure what to do here. There's a good chance this isn't the best behaviour. Looking for advice/opinions.

Reply:
The visible labels and ARIA labels look consistent with production, so this should be fine. We have passing tests that check the weekday labels are two characters long in German and Polish, so this edge case is covered :)